devtools/prepare-release.sh

```diff
@@ -196,6 +196,12 @@ the commits, tags and branch pointers before you push!
 For a major release you should be on the release-branch, not the master
 branch. While you can move around branch pointers later, it is a lot
 easier to branch first.
+
+A manual step needs to be run before release consist in checking
+documentation external links which can be done with the command:
+
+sphinx-build -M linkcheck docs/manuals/source docs/BareosMainReference -j2
```

Comment:
cmake is not available in our docbuild container. So I can't be used there, and you will have the run the complete command.
But it can still be added as suggestion for people doing the build in main environment.